src/GDAL.jl

```diff
@@ -30,7 +30,8 @@ module C
 end
 
 include("C/misc.jl")
-include("C/common.jl")
+include("types.jl")
+include("common.jl")
```

Comment:
I'm talking about the whole file.

I agree that quite a few of the constants are needed in normal use, but I can't help but feel they might be better off as symbols or immutables instead (makes them a lot more interpretable when rendered in the REPL, rather than becoming obscure constants). Hence my impression that the constants should be in `src/C/common.jl`, and the corresponding symbols/mappings (e.g. https://github.com/yeesian/GDALUtils.jl/blob/master/src/metadata.jl#L24) in `src/common.jl`.